lib/twilio-ruby/rest/api/v2010/account/call.rb

```diff
@@ -585,7 +585,7 @@ def initialize(version, payload, account_sid: nil, sid: nil)
                   'group_sid' => payload['group_sid'],
                   'parent_call_sid' => payload['parent_call_sid'],
                   'phone_number_sid' => payload['phone_number_sid'],
-                  'price' => payload['price'].to_f,
+                  'price' => payload['price'] == nil ? payload['price'] : payload['price'].to_f,
```

Comment:
May want to put this in a function in case there are other params that may have this issue; otherwise, I think the on-off is fine.

Reply:
It's a generated line. Other langs wrap it in a func because they're multi-lined.